### ISSUE TITLE
Systematization clarity

### ISSUE THREAD

**Issue by oetyng** [maidsafe/sn_dbc#271]
The recent change to tx naming went from `src_tx` and `dst_tx` to `spent_tx` and `dbc_creation_tx`.

There are a few issues with this.

The most obvious one is that there was a clear duality systematization expressed by using the two opposites of `src` and `dst` as suffixes.
The rationale behind it is that a dbc has a life time flow, where it comes from a _src tx_, and goes to a _dst tx_. That is the system design flow that is encapsulated in the naming. It is easy to understand what these two txs in `Spend` are in the frame of that life time flow. One is where the dbc came from, the other is where it went.

In comparison to that, there is not the same clear duality or pairing described by `spent_` and `dbc_creation_` suffixes.

Morover, we talk of **spent dbcs**, but not **spent txs**. A tx is a _transaction_, the word itself contains the fact that there has been spends, that the transaction irreversibly moves dbcs from one state to another (i.e. to being spent). To call a tx spent is like saying salsa sauce. However, a tx may be valid or not in the network, which is the existing terminology. Things are valid or not valid in the network.

There are additional issues though, with the specific name change.
The renaming was just making a small incision into the naming system, leaving other parts dangling. There are other places in the code base where src and dst are referred to.
Better than to do incomplete changes which just scrambles the whole terminology, is to create a complete systematization and switch atomically between them.

***

In summary, the change seems to have been haphazardly made with not enough thought put into it, and the suggestion is to go back to previous terminology, and to work on a complete systematization update to get a coherent and logical end result.
